app_feup/lib/controller/local_storage/app_shared_preferences.dart

```diff
@@ -83,6 +86,16 @@ class AppSharedPreferences {
     return Tuple2(userNum, userPass);
   }
 
+  /// Returns the user's faculties
+  static Future<List<String>> getUserFaculties() async {
+    final prefs = await SharedPreferences.getInstance();
+    final List<String> storedFaculties = prefs.getStringList(userFaculties);
+    if (storedFaculties == null) {
+      return [];
+    }
+    return storedFaculties;
```

Comment:
This can be a one-liner:
```suggestion
    return storedFaculties == null ? [] : storedFaculties;
```